### ISSUE TITLE
searchable operation gt, gte, lt, lte on AWSDate

### ISSUE THREAD

**Issue by aprilmintacpineda** [aws-amplify/amplify-cli#2775]
**Describe the bug**

`lte` and `gte` on `AWSDate` does not work with the following error:

![image](https://user-images.githubusercontent.com/21032419/69103044-3b593000-0a9f-11ea-8d9e-26f18fa7930f.png)

**To Reproduce**

1. Create a model with `@searchable`
2. Try to query with filter `gt`, `gte`, `lt`, `lte`
3. See error

-- or --

**Expected behavior**

The said searchable operations should be able to work on dates as well not just `Int`.

**Screenshots**

N/A

**Smartphone (please complete the following information):**

N/A

**Additional context**

N/A

**Sample code**

N/A

I kind of remember this working in the past, I even used `ge` and `le`. Did something change but I was unaware of? Would be cool if there's one place where we can go to see the change logs.

**Comment by SwaySway** [comment]
Closing this issue as the fix for this has been merged.

**Comment by aprilmintacpineda** [comment]
@SwaySway is there anything we need to do? Do we need to update @aws/amplify-cli?

**Comment by aprilmintacpineda** [comment]
This has been closed and marked as fixed but it still doesn't work.

```graphql
{
  searchTodos (
    limit: 999
    filter: {
      startedAt: {
        gt: "2020-02-02T00:00:00.000Z"
      }
    }
  ) {
    items {
      id
      title
      description
    }
  }
}
```

this will fail with:

```
Validation error of type WrongType: argument 'filter.startedAt'
contains a field not in 'SearchableStringFilterInput': 'gt' @ 'searchTodos'
```

And I did this on AppSync console.

**Comment by SwaySway** [comment]
Hello @aprilmintacpineda what version of the CLI are you using?
This fix came out in release 4.13.0 though we recommend using 4.13.1.
